tests/vision/base/test_context.py

```diff
@@ -93,84 +58,74 @@ def test_that_vision_context_raises_exception_for_unset_properties(mnist_dataset
             DeepchecksNotSupportedError,
             r'Check is irrelevant for Datasets without test dataset')
     )
-    assert_that(
-        calling(lambda: context.model),
-        raises(
-            DeepchecksNotSupportedError,
-            r'Check is irrelevant for Datasets without model')
-    )
 
 
 def test_empty_context_initialization():
     assert_that(
-        calling(Context).with_args(model_name="Name", ),
+        calling(Context).with_args(),
         raises(
             DeepchecksValueError,
-            r'At least one dataset \(or model\) must be passed to the method\!')
+            r'At least one dataset must be passed to the method\!')
     )
 
 
-def test_context_initialization_with_test_dataset_only(coco_test_visiondata):
+def test_context_initialization_with_test_dataset_only(coco_visiondata_test):
     assert_that(
-        calling(Context).with_args(model_name="Name", test=coco_test_visiondata),
+        calling(Context).with_args(test=coco_visiondata_test),
         raises(
             DatasetValidationError,
             r"Can't initialize context with only test\. if you have single dataset, "
             r"initialize it as train")
     )
 
 
-def test_context_initialization_with_train_dataset_only(coco_train_visiondata):
-    Context(model_name="Name", train=coco_train_visiondata)
-
-
-def test_context_initialization_with_model_only(mock_trained_mnist):
-    Context(model_name="Name", model=mock_trained_mnist)
-
-
-def test_context_initialization_with_training_model(mock_trained_mnist):
-    trained_mnist = copy.deepcopy(mock_trained_mnist.real_model)
-    trained_mnist.train()
-    assert_that(
-        calling(Context).with_args(model_name="Name", model=trained_mnist),
-        raises(
-            DatasetValidationError,
-            r'Model is not in evaluation state. Please set model training '
-            r'parameter to False or run model.eval\(\) before passing it.')
-    )
-
-
-def test_context_initialization_with_broken_model(mnist_dataset_train, mnist_dataset_test):
-    # Arrange
-    class BrokenModel(nn.Module):
-        def __call__(self, *args, **kwargs):
-            raise Exception("Invalid arguments")
+def test_context_initialization_with_train_dataset_only(coco_visiondata_train):
+    Context(train=coco_visiondata_train)  # should pass
 
-    model = BrokenModel()
-    model.eval()
 
-    # Act & Assert
-    assert_that(
-        calling(Context
-                ).with_args(train=mnist_dataset_train,
-                            test=mnist_dataset_test,
-                            model=model),
-        raises(
-            Exception,
-            r'Invalid arguments')
-    )
-
-
-def test_vision_context_helper_functions(mnist_dataset_train):
+# Currently, no model is supported in context
```

Comment:
Why not just delete?

Reply:
Seems like good logic I will forget existed when we will reintroduce model